### PR TITLE
Feature/aside

### PR DESCRIPTION
**Pull request by mayankrajendrat**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const heading=document.querySelector(".heading");
+heading.addEventListener("mouseover", function(event){
+   heading.style.fontSize="50px";
```
Comment (Akki2155) [inline script.js]: Please give relative unit

Comment (KNagaHariKrishna) [inline script.js]: Increase Font size

File: index.html [hunk 2]
```diff
@@ -19,5 +25,6 @@
             </aside>
         </section>
     </main>
+<script type="text/javascript" src="script.js"></script>
```
Comment (Akki2155) [inline index.html:28]: Please keep script file outside body

Comment (KNagaHariKrishna) [inline index.html:28]: Pls Add Out Side

**Review by KNagaHariKrishna — COMMENTED** [review]
Pls Add Out Side

**Review by Akki2155 — APPROVED** [review]

File: script.js (hunk 1 above)
Comment (prathmeshbabar1) [reply script.js]: Give colors

**Pull request merged**